pkg/piecestore/psserver/config.go

```diff
@@ -27,8 +27,8 @@ var (
 // Config contains everything necessary for a server
 type Config struct {
 	Path                   string        `help:"path to store data in" default:"$CONFDIR"`
-	AllocatedDiskSpace     int64         `help:"total allocated disk space, default(1GB)" default:"1073741824"`
-	AllocatedBandwidth     int64         `help:"total allocated bandwidth, default(100GB)" default:"107374182400"`
+	AllocatedDiskSpace     int64         `help:"total allocated disk space, default(1TiB)" default:"1099511627800"`
```

Comment:
maybe use `memory.Size` here instead and implement flag parsing so a user would be able to use `--allocated-disk-space 1TB`

Reply:
good idea

Reply:
Using https://github.com/cloudfoundry/bytefmt now

Reply:
See https://github.com/storj/storj/blob/master/internal/memory/sizes.go#L8, it should already support it.

Reply:
`AllocatedDiskSpace     memory.Size         `help:"total allocated disk space, default(1TiB)" default:"1TB"`, might do it

Reply:
done